### ISSUE TITLE
whether harbor support nfs or glusterfs to save data? 

### ISSUE THREAD

**Issue by httaotao** [goharbor/harbor#14347]
hello everyone, we are deploying harbor in an virtual mechine. we plan to deploy it into k8s cluster. but we want to know which pv is better for harbor and which one is not good, such as glusterfs, nfs  and so on . we know that  nexus is not support for glusterfs. thanks .

**Comment by zbh2021** [comment]
I  have try use  nfs this is valid way

**Comment by phin1x** [comment]
If you use nfs, you should read this issue first: https://github.com/goharbor/harbor/issues/13589#issuecomment-784042733

**Comment by httaotao** [comment]
> If you use nfs, you should read this issue first: [#13589 (comment)](https://github.com/goharbor/harbor/issues/13589#issuecomment-784042733)

thanks for answer. is there a gc problem when use nfs for blob?  the problem seems doesn't solve. maybe it is not a good way to use nfs now. thanks.

**Comment by zbh2021** [comment]
I'm not face this problem

**Comment by phin1x** [comment]
there is a patched image: `goharbor/registry-photon:v2.1.0-nfs`

**Comment by httaotao** [comment]
> there is a patched image: `goharbor/registry-photon:v2.1.0-nfs`

that's great ,thanks.

**Comment by phin1x** [comment]
@httaotao If you have the option to use some sort of s3 storage, I would recommend this

**Comment by httaotao** [comment]
> there is a patched image: `goharbor/registry-photon:v2.1.0-nfs`

sorry, I have a try to run this image, but I don't what I should do, when I run with common ``` sudo docker run -d --name harbor    goharbor/registry-photon:v2.1.0-nfs ```, then error happen:

```
Appending internal tls trust CA to ca-bundle ...
find: /etc/harbor/ssl: No such file or directory
Internal tls trust CA appending is Done.
configuration error: open /etc/registry/config.yml: no such file or directory

Usage: 
  registry serve <config> [flags]
Flags:
  -h, --help=false: help for serve


Additional help topics:
```
